src/geometry/triangle.cpp

```diff
@@ -0,0 +1,69 @@
+#include <geometry/triangle.h>
+#include <geometry/utils.h>
+#include <math.h>
+
+Triangle::Triangle() {
+    vertices[0] = Point();
+    vertices[1] = Point();
+    vertices[2] = Point();
+}
+
+Triangle::Triangle(const Point& v1, const Point& v2, const Point& v3) {
```

Comment:
```suggestion
Triangle::Triangle(Point v1, Point v2, Point v3) {
```

Queremos a cópia mesmo